.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report.
+title: '[Bug] <title>'
+labels: Bug, Needs Triage
+assignees: ''
+---
+
+<!--
+  To make it easier for us to help you — please follow the suggested format below.
+
+  Before opening a new issue, please search existing issues.
+-->
+
+# Bug Report
+
+## Relevant information
+
+<!-- Provide as much useful information as you can -->
+
+### Your Environment
+
+- Browser:
+- Browser version:
+- Cypress version:
+- nextjs-auth0 version:
+- cypress-nextjs-auth0 version:
+
+#### Steps to reproduce
+
+1. Step 1
+2. Step 2
+3. Step 3
+
+#### Observed Results
+
+<!-- What happened? This could be a description, log output, etc. -->
+
+#### Expected Results
```

Comment:
Do you think it's easier to answer these questions if expected behavior goes before actual behavior? For example "I expected to see the login screen appear. However, I keep getting redirected to the index page without being logged in."

Reply:
Yes, good idea. Take a look at https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/49/commits/e3ba11ed6a0029c63109fbba596738471f9dcfc1, please.